generators/spa/templates/Vue3JS/common/packages/Microsoft.AspNet.WebApi.Core.5.2.3/Content/web.config.transform

```diff
@@ -0,0 +1,11 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
```

Comment:
@alejoroman0605 Why isn't this using the same build files as the other module templates?